contracts/near/context-config/src/mutate.rs

```diff
@@ -183,7 +202,9 @@ impl ContextConfigs {
         for member in members {
             env::log_str(&format!("Added `{member}` as a member of `{context_id}`"));
 
-            let _ = ctx_members.insert(*member);
+            let _ignored = context.member_nonces.insert(*member, 0);
```

Comment:
this will then be a way to "reset the nonce" back to 0, we don't want to insert here for an existing member.